tests/main/uc20-create-partitions-encrypt/task.yaml

```diff
@@ -47,7 +47,7 @@ prepare: |
     echo "Get the UC20 gadget"
     if os.query is-focal; then
         snap download --channel=20/edge pc
-    elif os.query is-jammy; then
+    elif os.query is-jammy || os.query is-ubuntu 22.10; then
```

Comment:
(nitpick) maybe we should also change the first part of the `if` to the new syntax? i.e. instead of `os.query is-jammy` -> `os.query is-ubuntu 22.04` ? (same in the other places?)

Reply:
agree, thannks